src/main/java/commands/DeleteCommand.java

```diff
@@ -0,0 +1,27 @@
+package commands;
+
+import functional.DukeException;
+import functional.TaskList;
+import functional.Ui;
+import tasks.Task;
+
+
+public class DeleteCommand extends commands.Command {
```

Comment:
Any reason why you need to extends commands.Command rather than just extends Command?